pkg/model/api.go

```diff
@@ -51,6 +56,97 @@ func ImportModel(ctx context.Context, filePath, projectId, message, modelType, b
 		return err
 	}
 
-	log.Println("Starting import model job. JobId: ", importModelData.GetImportModelJobId())
+	importModelJobId := importModelData.GetImportModelJobId()
+	if waitForResults {
+		okStatus, job, err := waitForImportModelJob(ctx, importModelJobId)
+		if err != nil {
+			return err
+		}
+		if okStatus {
+			log.Println("Successfully imported model")
+			if mappingYaml != "" {
```

Comment:
can you explain why you print errors when mappingYaml is not empty?
```
// something like
someExplain := condition
```

Reply:
Updated